server/src/test/java/io/crate/integrationtests/TableFunctionITest.java

```diff
@@ -199,4 +202,29 @@ public void test_unnest_on_top_of_regexp_matches() throws Exception {
             "4.3.1\n"
         ));
     }
+
+    @UseJdbc(0)
```

Comment:
What's the reason for disabling JDBC here?

Reply:
oh sorry, was reusing my old commit and forgot to take them out.